data/plugins.json

```diff
@@ -20,11 +20,11 @@
     "node": 6
   },
   "transform-es2015-classes": {
-    "chrome": 46,
+    "chrome": 45,
     "edge": 13,
     "firefox": 45,
     "safari": 10,
-    "node": 5
+    "node": 4
```

Comment:
Is it possible to change spread to account for this? Or do we need to check if a user is using spread..